src/integrationTest/java/uk/gov/hmcts/reform/roleassignment/controller/WelcomeControllerIntegrationTest.java

```diff
@@ -57,5 +60,26 @@ public void welComeAPITest() throws Exception {
         assertEquals(
             "Service is unavailable:", 200, 200);
     }
+
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {"classpath:sql/insert_role_assignment_request.sql"})
+    public void shoudGetRecordCountFromRequestTable() throws Exception {
+        final int count = template.queryForObject(COUNT_RECORDS, Integer.class);
+        logger.info(" Total number of records fetched from role assignment request table...{}", count);
+        assertEquals(
+            "role_assignment_request record count ", 5, count);
```

Comment:
would this work in the PR and get 5 records?

Reply:
its embedded postgress database created on the fly and perform db operations. This test passed successfully in PR.